### ISSUE TITLE
[TW-1524] Build Broken

### ISSUE THREAD

**Issue by taskwarrior** [GothenburgBitFactory/taskwarrior#1550]
_Jack on 2015-01-26T04:06:51Z says:_ 

 Branch 2.4.1 is not building for me. According to git-bisect:

{code}
9bd9a3b285302da2d54ab847c20302c2719be9fe is the first bad commit
{code}

**Comment by taskwarrior** [comment]
Migrated metadata:
```
Created: 2015-01-26T04:06:51Z
Modified: 2015-02-07T19:35:11Z
```

**Comment by taskwarrior** [comment]
_Tomas Babej on 2015-01-26T07:30:57Z says:_ 

 I persume you have old GCC (< 4.8). Can you update?

**Comment by taskwarrior** [comment]
_Tomas Babej on 2015-01-26T07:35:19Z says:_ 

 FYI, this is how I made it work on Ubuntu 12 TLS:

{code}
sudo add-apt-repository ppa:ubuntu-toolchain-r/test
sudo apt-get update
sudo apt-get install g++-4.8
sudo update-alternatives --install /usr/bin/g++ g++ /usr/bin/g++-4.8 50
{code}

Taken from http://askubuntu.com/questions/522387/std-c11-not-recognized-on-gcc-4-8-1 .

**Comment by taskwarrior** [comment]
_Paul Beckingham on 2015-02-07T19:35:00Z says:_ 

 Ordinarily a broken build is no cause for alarm, this is a development branch after all.

This case is different.  This is the first step towards making use of C++11 in the code base, and requires gcc 4.4 (released 2009-04-21) or clang 2.9 (released 2011-04-06), which are the versions where C++11 N1984 is first supported.
